lib/Onyx.js

```diff
@@ -847,6 +846,11 @@ function evictStorageAndRetry(error, onyxMethod, ...args) {
  * @returns {Promise}
  */
 function set(key, value) {
+    if (_.isNull(value)) {
+        Logger.logInfo(`Removing the key: ${key} as it called set() with null value`);
```

Comment:
I think we should remove this log line, and also the log a couple of lines below. I don't think they are useful at all.

Reply:
>also the log a couple of lines below
To confirm. Do you mean this line

'Logger.logInfo(`set() called for key: ${key}${_.isObject(value) ? ` properties: ${_.keys(value).join(',')}` : ''}`);'  850/854? 

I added the above log line w.r.t the current log line present 850/854. I can remove both if they are not required.

Reply:
Removed the log lines and added a new commit

Reply:
Yes, let's please remove both. I've been meaning to remove that existing log line for a while.

Reply:
Done. Removed the log lines with the latest commit.